versions.cfg

```diff
@@ -29,6 +29,7 @@ colorama = 0.4.3
 configparser = 4.0.2
 contextlib2 = 0.6.0.post1
 coverage = 5.0.3
+distlib = 0.3.0
```

Comment:
Is this package needed by Chameleon? Than it should be in `versions-prod.cfg` as that file contains all the pins needed to run a Zope instance. The ones here are needed for running the tests etc.

Reply:
This was pulled in by a third party package

Reply:
See a268a86f6abecea1e478217d13a98b61554d914f